tests/Cargo.toml

```diff
@@ -6,6 +6,7 @@ edition = "2021"
 [dependencies]
 coffee_lib = { path = "../coffee_lib" }
 coffee_testing = { path = "../coffee_testing" }
+coffee_httpd = { path = "../coffee_httpd" }
```

Comment:
Mh, I am thinking, as model we can use what we have already inside the coffee_lib? https://github.com/coffee-tools/coffee/blob/master/coffee_lib/src/types/mod.rs

Maybe make two modules as `request` and `response`?